src/main/java/net/dv8tion/jda/api/entities/ICategorizableChannel.java

```diff
@@ -21,18 +21,37 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
-//TODO-v5: Need Docs
+
+/**
+ * Represents a channel that can be a member of a {@link Category}.
+ * Channels represented by this interface can have a parent {@link Category}.
+ *
+ * @see Category
+ * @see Guild#getCategories()
+ */
 public interface ICategorizableChannel extends GuildChannel, IPermissionContainer
 {
     //TODO-v5: Docs
     @Override
     @Nonnull
     ICategorizableChannelManager<?, ?> getManager();
 
-    //TODO-v5: Docs
+    /**
+     * Get the snowflake of the {@link Category} that contains this channel.
+     *
+     * This will return {@code 0} if this channel doesn't have a parent category.
+     *
+     * @return The Discord ID snowflake of the parent channel as a long.
+     */
     long getParentCategoryIdLong();
 
-    //TODO-v5: Docs
+    /**
+     * Get the snowflake of the {@link Category} that contains this channel.
+     *
+     * This will return {@code 0} if this channel doesn't have a parent category.
+     *
+     * @return The Discord ID snowflake of the parent channel as a String.
+     */
     default String getParentCategoryId()
     {
         return Long.toUnsignedString(getParentCategoryIdLong());
```

Comment:
```suggestion
    /**
     * Get the snowflake of the {@link net.dv8tion.jda.api.entities.Category Category} that contains this channel.
     *
     * This will return {@code null} if this channel doesn't have a parent category.
     *
     * @return Possibly-null String representation of the Discord ID snowflake of the parent channel.
     */
     @Nullable
    default String getParentCategoryId()
    {
        long parentID = getParentCategoryIdLong();
        if (parentID == 0L)
            return null;
        return Long.toUnsignedString(parentID);
```